### PR TITLE
tidy up the code

### PR DESCRIPTION
**Pull request by timogoebel**

### CODE REVIEW

File: foreman_build_history.gemspec [hunk 1]
```diff
@@ -7,15 +7,14 @@ Gem::Specification.new do |s|
   s.date        = Date.today
   s.authors     = ['Shim Shtein']
   s.email       = ['shteinshim@gmail.com']
-  s.homepage    = 'http://github.com'
-  s.summary     = 'Summary of ForemanBuildHistory.'
+  s.homepage    = 'http://github.com/theforeman/foreman_build_history'
```
Comment (ShimShtein) [inline foreman_build_history.gemspec]: In the meantime it's in my repo, `https://github.com/ShimShtein/foreman_build_history`.
Unless you already want to move it to core ;)

File: README.md [hunk 2]
```diff
@@ -2,10 +2,16 @@
...
+
+| Foreman Version | Plugin Version |
+| --------------- | -------------- |
+| >= 1.12         | any            |
+
 ## Installation
 
-See [How_to_Install_a_Plugin](http://projects.theforeman.org/projects/foreman/wiki/How_to_Install_a_Plugin)
-for how to install Foreman plugins
+See [Plugins install instructions](https://theforeman.org/plugins/)
```
Comment (ShimShtein) [inline README.md]: This plugin is not packaged, the only way to install it right now is through gems. Can we change the URL to more specific one: `https://theforeman.org/plugins/#2.3AdvancedInstallationfromGems`?

File: lib/foreman_build_history/facet.rb [hunk 3]
```diff
@@ -1,7 +1,2 @@
 Facets.register(BuildHistoryFacet) do
```
Comment (ShimShtein) [inline lib/foreman_build_history/facet.rb]: We can get rid of the `do..end` block entirely.
At least until we want to add a per-host history tab.

**Review by ShimShtein — CHANGES_REQUESTED** [review]
Thank you for tidying up the plugin a bit, I really appreciate it! If you don't mind, I have some change proposals inline.

File: foreman_build_history.gemspec (hunk 1 above)
Comment (timogoebel) [reply foreman_build_history.gemspec]: I do suggest moving it to the foreman.org - it increased visibility and it's easier to get jenkins integration. In addition permissions management is better.

But I'll change this for now.

File: README.md (hunk 2 above)
Comment (timogoebel) [reply README.md]: I do want to package it once it's ready for a release.

File: lib/foreman_build_history/facet.rb (hunk 3 above)
Comment (timogoebel) [reply lib/foreman_build_history/facet.rb]: Can you already add a tab to the host show page through facets? I tried with pagelets but the required mount points were missing as well.

Comment (ShimShtein) [reply lib/foreman_build_history/facet.rb]: Not exactly through facets, but through initializer:
https://github.com/theforeman/foreman/blob/develop/config/initializers/puppet.rb#L3

File: foreman_build_history.gemspec (hunk 1 above)
Comment (ShimShtein) [reply foreman_build_history.gemspec]: I am all for it, but I would like to add proper testing and rubocop before we do it.

**Comment by timogoebel** [comment]
@ShimShtein : Thanks, I have addressed your comments.

**Review by ShimShtein — APPROVED** [review]

**Comment by ShimShtein** [comment]
Thanks @timogoebel!

Merged as bd81a64b9baa70568a537687bf603f1727ff2226

**Pull request closed without merging**